### PR TITLE
Update .htaccess

### PR DESCRIPTION
**Pull request by turbomam**
confirmed RewriteRule ^(.*)$      https://microbiomedata.github.io/nmdc-schema/$1 [R=302,L]

### CODE REVIEW

File: nmdc/.htaccess [hunk 1]
```diff
@@ -6,7 +6,9 @@ RewriteEngine on
 
 RewriteRule ^(.*).yaml$ https://raw.githubusercontent.com/microbiomedata/nmdc-schema/main/src/schema/$1.yaml [R=302,L]
 
+# works
```
Comment (davidlehn) [inline nmdc/.htaccess:9]: One would hope so!  Is this comment needed?

File: nmdc/.htaccess [hunk 2]
```diff
@@ -6,7 +6,9 @@ RewriteEngine on
...
 RewriteRule ^(.*)$      https://microbiomedata.github.io/nmdc-schema/$1 [R=302,L]
+# eg https://w3id.org/nmdc/Biosample -> https://microbiomedata.github.io/nmdc-schema/Biosample/
```
Comment (davidlehn) [inline nmdc/.htaccess:11]: There's no trailing slash: https://microbiomedata.github.io/nmdc-schema/Biosample

Comment (TallTed) [reply nmdc/.htaccess:11]:
```suggestion
# e.g., https://w3id.org/nmdc/Biosample -> https://microbiomedata.github.io/nmdc-schema/Biosample
```

**Comment by dgarijo** [comment]
@turbomam any updates for this PR?

**Comment by turbomam** [comment]
Thanks for the reminder. I will either add some content or close it by the end of the week.

**Comment by dgarijo** [comment]
Ping @turbomam

**Pull request closed without merging**